user_tools/src/spark_rapids_pytools/cloud_api/dataproc.py

```diff
@@ -55,22 +56,30 @@ def _set_remaining_configuration_list(self) -> None:
                                                           'confProperties',
                                                           'propertiesMap')
         if properties_map_arr:
-            config_cmd_prefix = ['gcloud', 'config', 'get']
-            for prop_entry in properties_map_arr:
-                prop_entry_key = prop_entry.get('propKey')
-                if self.ctxt.get(prop_entry_key):
-                    # Skip if the property already set
-                    continue
-                prop_cmd = config_cmd_prefix[:]
-                section_entry = prop_entry.get('section')
-                prop_cmd.append(f'{section_entry}/{prop_entry_key}')
-                cmd_args = {
-                    'cmd': prop_cmd,
-                }
-                prop_cmd_obj = SysCmd().build(cmd_args)
-                prop_cmd_res = prop_cmd_obj.exec()
-                if prop_cmd_res:
-                    self.ctxt.update({prop_entry_key: prop_cmd_res})
+            # We support multiple CLI configurations, the following two dictionaries map
```

Comment:
After these changes, it seems that the overridden function has same structure as the `_set_remaining_configuration_list()` from the base class. Can we use the base implementation if this is not needed?

Reply:
Yes, they are similar, but the underlying dictionaries to store the config section/file/keys are actually very different. For instance, `config_section_file` in `dataproc` is a reverse map of `config_file_section` in the base class.

Reply:
I see. Thanks Cindy.